src/Components/ADT3DSceneBuilder/Internal/Behaviors/Internal/StatusTab.tsx

```diff
@@ -95,7 +96,8 @@ const StatusTab: React.FC<IStatusTabProps> = ({ onValidityChange }) => {
                             draft.visuals.splice(index, 1);
                         }
                         statusVisual[propertyName as any] = value as any;
-                    } else {
+                    } else if (value !== '') {
```

Comment:
do we want to specifically handle this empty string case? or should we handle more braodly to not do this in the cases of undefined/null as well?

Reply:
Value is typed as a string here so I think it's fine, but if ya wanna be extra safe, you could use

`else if(value)`